src/common/selftest/selftest_heater.cpp

```diff
@@ -62,33 +160,36 @@ bool CSelftestPart_Heater::Loop() {
             m_TempCount = 0;
             return true;
         }
-        setTargetTemp(m_pConfig->target_temp);
+        setTargetTemp(m_config.target_temp);
         m_Time = Selftest.m_Time;
         m_MeasureStartTime = m_Time;
         m_Temp = 0;
         m_TempCount = 0;
-        break;
+        break;*/
     }
     case spsMeasure:
-        if ((Selftest.m_Time - m_Time) < TEMP_MEASURE_CYCLE_DELAY) {
+        if (int(m_EndTime - Selftest.m_Time) > 0) {
+            m_Temp = getTemp();
+            return true;
+        }
+
+        /* if ((Selftest.m_Time - m_Time) < TEMP_MEASURE_CYCLE_DELAY) {
```

Comment:
what about this commented code? Is it still relevant?

Reply:
it is, did not want to remove Roberts code